packages/content-type-read-receipt/README.md

```diff
@@ -1 +1,107 @@
-## Coming soon
+# Read receipt content type
+
+This package provides an XMTP content type to support read receipts to messages.
+
+> **Open for feedback**
+> You are welcome to provide feedback on this implementation by commenting on the [Read Receipts content type proposal](https://github.com/xmtp/XIPs/pull/27).
+
+## What’s a read receipt?
+
+A read receipt is a message sent to confirm that a previously sent message has been read by the recipient. With XMTP, read receipts are special messages with the `ReadReceipt` content type. They contain a timestamp of when the original message was read.
+
+When someone receives a message using an app with read receipts enabled, their XMTP client can send a read receipt when they open that message.
+
+## Why read receipts?
+
+Read receipts give the sender confirmation that the recipient has read their message. This avoids uncertainty about whether a message was seen, without needing to rely on a manual response.
+
+## Install the package
+
+```bash
+# npm
+npm i @xmtp/content-type-read-receipt
+
+# yarn
+yarn add @xmtp/content-type-read-receipt
+
+# pnpm
+pnpm i @xmtp/content-type-read-receipt
+```
+
+## Create a read receipt
+
+With XMTP, read receipts are represented as objects with the following keys:
+
+- `timestamp`: The timestamp the read receipt was sent, in ISO 8601 format
+
+```tsx
+const readReceipt: ReadReceipt = {
+  timestamp: new Date().toISOString();
+};
+```
+
+## Send a read receipt
+
+If a sender has opened a conversation and has not yet sent a read receipt for its received messages (this can either be done with each message or the most recent message and is an individual app decision), you can send a read receipt like so:
+
+```tsx
+await conversation.messages.send(
+  {
+    timestamp: new Date().toISOString(),
+  },
+  ContentTypeReadReceipt,
+);
+```
+
+## Receive a read receipt
+
+Now that you can send a read receipt, you can also receive a read receipt that was sent from another user. For example:
+
+```tsx
+// Assume `loadLastMessage` is a thing you have
+const message: DecodedMessage = await loadLastMessage();
+
+if (message.contentType.sameAs(ContentTypeReadReceipt)) {
+  // We have a read receipt
+  return;
+}
+```
+
+## Display the read receipt
+
+Generally, read receipts should be displayed under the message it's associated with, and can include a timestamp or no timestamp. Ultimately, how you choose to display read receipts is completely up to you.
+
+## UX best practices
+
+While this is a per-app decision, the best practice is to provide users with the option to opt out of sending read receipts. If a user opts out, when they read a message, a read receipt will not be sent to the sender of the message.
```

Comment:
@daria-github reworded this a bit - did I get it right? =)

Reply:
yep, nailed it, thanks for making some verbiage changes directly in here - appreciate you @jhaaaa! 🙏